CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+# Version 3.3.0-BETA
+* Update OpenTelemetry to 1.14 [#2257](https://github.com/microsoft/ApplicationInsights-Java/pull/2257).
+* Deprecate logs' LoggingLevel custom property [#2254](https://github.com/microsoft/ApplicationInsights-Java/pull/2254).
+* Add support for OpenTelemetry log exporter and metric exporter.
```

Comment:
users won't see a difference related to logs (there is no logs API, only auto-instrumentation + logs SDK)
```suggestion
* Add support for OpenTelemetry metrics API
```